modules/ROOT/pages/_partials/installing-mobile-services.adoc

```diff
@@ -2,162 +2,160 @@
-
-NOTE: {org-name} recommends running OpenShift using the method described in this document. However, there are many ways to run OpenShift, and {product-name} is compatible with most of these methods. There is a known issue with Minishift that makes it unsuitable for mobile development. This issue is described in link:https://github.com/minishift/minishift/issues/1287[Issue 1287, window="_blank"].
+To use {product-name}, you must have OpenShift instance and run installation script which enables {product-name}. For more information about OpenShift, see link:https://www.openshift.org/[OpenShift website, window="_blank"].
 
 [[prerequisites]]
 == Prerequisites
 
-* MacOS or Linux
-+
-NOTE: Due to an link:https://github.com/openshift/origin/issues/18747[OpenShift bug], {product-name} will not run on RHEL/CentOS 7.4 or RHEL/CentOS 7.5.
-
-* A system running OpenShift `oc cluster up` as described in link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#prerequisites[Local Cluster Management, window="_blank"]
-+
-NOTE: Installation on macOS with Docker Machine has not been tested, use the link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#macos-with-docker-for-mac[Docker for Mac] instructions when installing OpenShift.
+* OpenShift 3.11 instance
 +
-** Use link:https://www.openshift.org/download.html[OpenShift client tools, window="_blank"] version 3.9
+** This instace must have service catalog, ansible service broker and template service broker services installed.
 +
-IMPORTANT: OpenShift oc executable must be located in a system location that is known to all shells (e.g. /usr/local/bin)
+** You need to have full access to this OpenShift instance, i.e. user with `cluster-admin` privileges.
 +
-IMPORTANT: Ensure that you can run link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#getting-started[oc cluster up, window="_blank"] with no errors before moving on to the installation of AeroGear.
-+
-* Access to a link:https://hub.docker.com/[Docker Hub] account. The installer uses Docker Hub as a source for AeroGear Docker images.
-//** For Linux (RHEL), manage Docker as a non-root user. For more information, see section _Manage Docker as a non-root user_ in  link:https://docs.docker.com/install/linux/linux-postinstall/[Post-installation steps for Linux].
-
-* link:https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[Ansible, window="_blank"] (version 2.6 or above)
+NOTE: If you don't have OpenShift instance you can use `oc cluster up` or Minishift to xref:local-setup[setup OpenShift locally].
 
-* For Linux (Fedora), add an extra port to the `dockerc` zone:
+* link:https://www.openshift.org/download.html[OpenShift client tools, window="_blank"] version 3.11
 
-[source,sh]
-----
-$ firewall-cmd --permanent --zone dockerc --add-port 443/tcp
-$ firewall-cmd --reload
-----
+* link:https://www.docker.com/[Docker]
```

Comment:
which version?

Reply:
The installation script uses `mikefarah/yq` docker image. It's documentation does not say what is the minimal supported version of docker. But will probably work with any version.

Reply:
well, oc cluster up requires a specific version i believe

Reply:
that is right, but there is no need for `oc cluster up` to setup mobile services on OpenShift